### ISSUE TITLE
Crawl Issue: Missing Alt Text

### ISSUE THREAD

**Issue by shsteimer** [hlxsites/uhg-surest#150]
Dawit to provide list of pages/images where missing

**Comment by shsteimer** [comment]
Closing this out as completed. Will follow up after launch to have their authoring check/update alt text
